index.js

```diff
@@ -0,0 +1,162 @@
+contactMenu.append(contactLink);
+navMenuDiv.append(contactMenu);
+navBar.append(navMenuDiv);
+navBar.prepend(navHeading);
+const currentDate = DateTime.now();
+
+document.querySelector('.time').innerHTML = currentDate.toLocaleString(DateTime.DATETIME_MED);
+
+class Book {
+  constructor() {
+    this.bobject = {};
+    this.list = [];
+  }
+
+  add(title, author) {
+    this.bobject = { title, author };
+    this.list.push(this.bobject);
+    localStorage.setItem('data', JSON.stringify(this.list));
+  }
+
+  remove(getId) {
+    let new1 = 0;
+    while (new1 < this.list.length) {
+      if (new1.toString() === getId) {
+        this.list.splice(new1, 1);
+        localStorage.setItem('data', JSON.stringify(this.list));
+      }
+
+      new1 += 1;
+    }
+  }
+}
+
+const book = new Book();
+
+const bookList = document.createElement('ul');
+bookList.style.marginTop = '20px';
+
+bookList.className = 'book-list';
+displayPage.append(bookList);
+const anyRandomNAme = () => {
+  book.list.forEach((each, bookId) => {
+    const list1 = document.createElement('li');
+    list1.className = 'list';
+
+    const title = document.createElement('h2');
+    title.innerHTML = `"${each.title}"`;
+    const phrase = document.createElement('b');
+    phrase.textContent = 'By';
+    list1.appendChild(title);
+    list1.appendChild(phrase);
+
+    const author = document.createElement('h2');
+    author.innerHTML = each.author;
+    list1.appendChild(author);
+
+    const removeButton = document.createElement('button');
+    removeButton.className = 'remove';
+    removeButton.id = `${bookId}`;
+    removeButton.innerHTML = 'Remove';
+    list1.appendChild(removeButton);
+
+    bookList.appendChild(list1);
+    removeButton.addEventListener('click', function removeBtnHandler() {
+      const getId = this.id;
+      book.remove(getId);
+      bookList.innerHTML = '';
+      anyRandomNAme();
+    });
+  });
+};
+
+// SPA implementation
+
+function listBook() {
+  displayPage.style.display = 'flex';
+  displayPage.style.flexDirection = 'column';
+  document.querySelector('.add-page').style.display = 'none';
+  document.querySelector('.contact-me').style.display = 'none';
+}
+window.addEventListener('load', () => {
+  listBook();
+});
+listLink.addEventListener('click', () => {
+  listBook();
+});
+addLink.addEventListener('click', (e) => {
+  e.preventDefault();
+  displayPage.style.display = 'none';
+  document.querySelector('.add-page').style.display = 'flex';
+  document.querySelector('.contact-me').style.display = 'none';
+});
+
+contactLink.addEventListener('click', (e) => {
+  e.preventDefault();
+  displayPage.style.display = 'none';
+  document.querySelector('.add-page').style.display = 'none';
+  document.querySelector('.contact-me').style.display = 'flex';
+});
+
+// const formHeading = document.createElement('h1');
+// formHeading.className = 'form-heading';
+// formHeading.textContent = 'All Awesome Books';
+
+// section.prepend(formHeading);
```

Comment:
- [ ] Please remove all the commented code from your codebase in order to make your code clean :+1: